b2sharedb_db_create.sql

```diff
@@ -0,0 +1,1733 @@
+--
+
+CREATE INDEX ix_access_actionsroles_argument ON access_actionsroles USING btree (argument);
+
+
+--
+-- Name: ix_access_actionsusers_action; Type: INDEX; Schema: public; Owner: b2share
+--
+
+CREATE INDEX ix_access_actionsusers_action ON access_actionsusers USING btree (action);
+
+
+--
+-- Name: ix_access_actionsusers_argument; Type: INDEX; Schema: public; Owner: b2share
+--
+
+CREATE INDEX ix_access_actionsusers_argument ON access_actionsusers USING btree (argument);
+
+
+--
+-- Name: ix_files_object__mimetype; Type: INDEX; Schema: public; Owner: b2share
+--
+
+CREATE INDEX ix_files_object__mimetype ON files_object USING btree (_mimetype);
+
+
+--
+-- Name: ix_oaiserver_set_name; Type: INDEX; Schema: public; Owner: b2share
+--
+
+CREATE INDEX ix_oaiserver_set_name ON oaiserver_set USING btree (name);
+
+
+--
+-- Name: ix_oauth2server_client_client_secret; Type: INDEX; Schema: public; Owner: b2share
+--
+
+CREATE UNIQUE INDEX ix_oauth2server_client_client_secret ON oauth2server_client USING btree (client_secret);
+
+
+--
+-- Name: ix_oauth2server_token_access_token; Type: INDEX; Schema: public; Owner: b2share
+--
+
+CREATE UNIQUE INDEX ix_oauth2server_token_access_token ON oauth2server_token USING btree (access_token);
+
+
+--
+-- Name: ix_oauth2server_token_refresh_token; Type: INDEX; Schema: public; Owner: b2share
+--
+
+CREATE UNIQUE INDEX ix_oauth2server_token_refresh_token ON oauth2server_token USING btree (refresh_token);
+
+
+--
+-- Name: ix_records_metadata_version_end_transaction_id; Type: INDEX; Schema: public; Owner: b2share
+--
+
+CREATE INDEX ix_records_metadata_version_end_transaction_id ON records_metadata_version USING btree (end_transaction_id);
+
+
+--
+-- Name: ix_records_metadata_version_operation_type; Type: INDEX; Schema: public; Owner: b2share
+--
+
+CREATE INDEX ix_records_metadata_version_operation_type ON records_metadata_version USING btree (operation_type);
+
+
+--
+-- Name: ix_records_metadata_version_transaction_id; Type: INDEX; Schema: public; Owner: b2share
+--
+
+CREATE INDEX ix_records_metadata_version_transaction_id ON records_metadata_version USING btree (transaction_id);
+
+
+--
+-- Name: ix_transaction_user_id; Type: INDEX; Schema: public; Owner: b2share
+--
+
+CREATE INDEX ix_transaction_user_id ON transaction USING btree (user_id);
+
+
+--
+-- Name: uidx_type_pid; Type: INDEX; Schema: public; Owner: b2share
+--
+
+CREATE UNIQUE INDEX uidx_type_pid ON pidstore_pid USING btree (pid_type, pid_value);
+
+
+--
+-- Name: useridentity_id_user_method; Type: INDEX; Schema: public; Owner: b2share
+--
+
+CREATE UNIQUE INDEX useridentity_id_user_method ON oauthclient_useridentity USING btree (id_user, method);
+
+
+--
+-- Name: access_actionsroles fk_access_actionsroles_role_id_accounts_role; Type: FK CONSTRAINT; Schema: public; Owner: b2share
+--
+
+ALTER TABLE ONLY access_actionsroles
+    ADD CONSTRAINT fk_access_actionsroles_role_id_accounts_role FOREIGN KEY (role_id) REFERENCES accounts_role(id);
```

Comment:
As said above, can you check that this is the constraint name which was used in B2SHARE v2.0.0